azure-iot-hub/azure/iot/hub/iothub_registry_manager.py

```diff
@@ -192,6 +283,11 @@ def update_device_with_x509(
         :param str secondary_thumbprint: Secondary X509 thumbprint.
         :param str status: Initital state of the created device.
             (Possible values: "enabled" or "disabled").
+        :param bool iot_edge: Whether or not the created device is an IoT Edge device.
```

Comment:
I see several of these methods have added a param for IoTEdge. Are we sure this is correct? - Edge with X509 seems odd to me, but I don't know the spec.

Reply:
Edge with X509 was already there in the SDK for the _create_ functions, I just added it to the _update_ functions. And Anca Antochi from platform said that updating iotEdge is supported by the service.